lib/client-assets.php

```diff
@@ -99,66 +99,84 @@ function gutenberg_register_scripts_and_styles() {
 
 	register_tinymce_scripts();
 
+	wp_register_script(
+		'wp-polyfill',
+		null,
+		array(
+			'wp-polyfill-ecmascript',
+		)
+	);
+	wp_script_add_data(
+		'wp-polyfill',
+		'data',
+		gutenberg_get_script_polyfill(
+			array(
+				'\'fetch\' in window' => 'wp-polyfill-fetch',
+				'document.contains'   => 'wp-polyfill-node-contains',
+				'window.FormData && window.FormData.prototype.keys' => 'wp-polyfill-formdata',
```

Comment:
Weren't you planning to add element closest or are you leaving this for a separate PR?

Reply:
>Weren't you planning to add element closest or are you leaving this for a separate PR?

If we can review and merge #9750 before this one, I can rebase and add it.

Reply:
Seems like the opposite is simpler right?